### PR TITLE
Add the 0b1010 commandments

### PR DESCRIPTION
**Pull request by mattjegan**

### CODE REVIEW

File: scripture.txt [hunk 1]
```diff
@@ -1,3 +1,14 @@
 And the Scrummaster said, the developer shall go up: behold, I have delivered the user stories into the board.
 And in their terminal Git shall bear thee up, lest at any time thou stash thy commits against a repo.
 Therefore being justified by tests, we have peace with merges.
+Thou shall obey the 0b1010 commandments.
+You shall have no other branchs before Master.
```
Comment (kennethlove) [inline scripture.txt]: spelling, should be "branches"

File: scripture.txt [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+You shall not make bugs.
+You shall not take the name of the Linus your BDFL in vain.
```
Comment (kennethlove) [inline scripture.txt]: I'd rather not name individuals (also Linus isn't a good tech person).

**Review by kennethlove — CHANGES_REQUESTED** [review]
Can you swap the "You"s with "Thou"s, etc?

**Comment by mattjegan** [comment]
@kennethlove No worries, made the changes and updated "your" to "thy" as well.

**Comment by mattjegan** [comment]
@kennethlove could we get this merged?

**Comment by mattjegan** [comment]
Cheers 👍

**Pull request merged**